web/public/js/directives/user/user-actions/template.dust

```diff
@@ -0,0 +1,28 @@
+<div uib-dropdown is-open="status.isopen"ng-if="cdUA.canEdit" class="cd-user-actions hidden-xs hidden-sm" ng-if="loggedInUser && ((loggedInUser.id === profile.userId) || profile.myChild">
+  <button id="cd-dojo-action" type="button" class="btn btn-secondary" uib-dropdown-toggle ng-disabled="disabled">
+    <i class="fa fa-gear"></i> <span class="caret"></span>
+  </button>
+  <ul class="dropdown-menu" uib-dropdown-menu role="menu" aria-labelledby="cd-dojo-action">
+    <li role="menuitem">
+      <a ui-sref="edit-user-profile({userId: cdUA.user.userId})" ng-if="cdUA.currentState !== 'edit-user-profile'" alt="{@i18n key="Edit Profile"/}">
+        <i class="fa fa-edit"></i>
+        {@i18n key="Edit Profile"/}
+      </a>
+    </li>
+    <li role="menuitem">
+      <a ui-sref="user-profile({userId: cdUA.user.userId})" target="_blank" ng-if="cdUA.currentState !== 'user-profile'" alt="{@i18n key="Profile"/}">
```

Comment:
this button seems more like a save ansd go back to my profile button, maybe change it to preview ?